src/Patient.java

```diff
@@ -0,0 +1,34 @@
+import java.util.ArrayList;
+
+import components.standard.Standard;
+
+/**
+ * Patient kernel component with primary methods.
+ *
+ * @author A. Ghastine
+ */
+public interface Patient extends Standard<Patient> {
```

Comment:
If this is just meant to store data, I might recommend looking into Java records. It would save you a lot of time.